index.js

```diff
@@ -2,11 +2,13 @@ const feature1 = require('./feature1');
 const feature2 = require('./feature2');
 const feature3 = require('./feature3');
 const feature4 = require('./feature4');
+const feature6 = require('./feature6');
 // hey
 
 module.exports = {
   feature1,
   feature2,
   feature3, // no there is no problem
-  feature4
+  feature4,
+  feature6
```

Comment:
```suggestion
  feature6,
```
use a trailing comma

Reply:
Applied! Thank you